great_expectations/datasource/data_connector/inferred_asset_glue_catalog_data_connector.py

```diff
@@ -0,0 +1,194 @@
+import logging
+from typing import Any, Dict, Iterator, List, Optional, Tuple
+
+from great_expectations.datasource.data_connector.configured_asset_glue_catalog_data_connector import (
+    ConfiguredAssetGlueCatalogDataConnector,
+)
+from great_expectations.exceptions import DataConnectorError
+from great_expectations.execution_engine import ExecutionEngine
+
+logger = logging.getLogger(__name__)
+
+
+class InferredAssetGlueCatalogDataConnector(ConfiguredAssetGlueCatalogDataConnector):
```

Comment:
The original architecture intended for both Inferred and Configured DataConnector classes -- in general -- to inherit from the same class.  Could that be achieved here as well, or is there a compelling critical advantage for having `InferredAssetGlueCatalogDataConnector` class extend the `ConfiguredAssetGlueCatalogDataConnector` class?

Reply:
It can be achieved but some methods, like the build_batch_spec, will have the same code. The **ConfiguredAssetGlueCatalogDataConnector** class inherits from the **ConfiguredAssetSqlDataConnector** because the methods are pretty much the same. Would you prefer to not have this dependency?

Reply:
@lccasagrande Should we use just "Glue" in class names or "AmazonGlue" or "AWSGlue"?  Please let me know which is best.  Thank you.